src/services/gateways/aws.ts

```diff
@@ -47,6 +47,10 @@ import {
   DescribeNetworkInterfacesCommand,
   EC2,
   Tag,
+  AllocateAddressCommand,
+  DescribeAddressesCommand,
+  ReleaseAddressCommand,
+  AllocateAddressCommandInput,
 } from '@aws-sdk/client-ec2'
```

Comment:
can we use `this.EC2` which doesn't require these imports and then instantiating the objects?

Reply:
ohh I can take a look at it 👍🏼